libmamba/include/mamba/core/util.hpp

```diff
@@ -215,7 +215,7 @@ namespace mamba
         // Returns 0 if `is_locked() == false`.
         std::size_t count_lock_owners() const
         {
-            return impl.has_value() ? impl.value().use_count() : 0;
+            return std::size_t(impl.has_value() ? impl.value().use_count() : 0);
```

Comment:
I prefer to keep the explicit instantiation, this way it is easier to see that we have potentially the wrong type during a later refactoring

Reply:
What about the following, that way we still get an error in case of narrowing during a future refactoring.
```suggestion
            return std::size_t{impl.has_value() ? impl.value().use_count() : 0};
```